### PR TITLE
Upgrade to Okta Spring Boot 2.1.6

### PR DESCRIPTION
**Pull request by arvindkrishnakumar-okta**

### CODE REVIEW

File: start-site/src/main/resources/application.yml [hunk 1]
```diff
@@ -522,7 +522,7 @@ initializr:
             - compatibilityRange: "[2.4.1,2.5.0-M1)"
               version: 2.0.1
             - compatibilityRange: "[2.5.0-M1,2.7.0-M1)"
```
Comment (mraible) [inline start-site/src/main/resources/application.yml:524]: I've had success using the Okta Spring Boot starter with Spring Boot 3. Is there any reason not to increase this compatibility range?

**Comment by mraible** [comment]
@snicoll Can you please help with this?

File: start-site/src/main/resources/application.yml (hunk 1 above)
Comment (snicoll) [reply start-site/src/main/resources/application.yml:524]: We don't generally think it's a good idea for a single version of a dependency to be use with both Spring Boot 2 and Spring Boot 3.

Comment (snicoll) [reply start-site/src/main/resources/application.yml:524]: However, it's a bit strange that the entry is still blocked for Spring Boot 2.7 that GA for months now...

Comment (snicoll) [reply start-site/src/main/resources/application.yml:524]: Given https://github.com/okta/okta-spring-boot/blob/7c9742fb5fee90c38fb2afc947f7b67047bb1dd6/pom.xml#L34 I am going to increase the range...

**Pull request closed without merging**